src/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -34,6 +34,7 @@ function MoveItemProperties() {
       '          {{vm.item.name}}',
       '        </a>',
       '      </div>',
+      '      <i translate-attr="{ title: \'DtrReview.DiscussHint\'}" ng-if="vm.item.talentGrid && vm.item.equipment && vm.item.lockable" class="dtr-review-discuss-button fa fa-comment" ng-click="vm.openDiscuss()"></i>',
```

Comment:
We should introduce a `reviewable` property rather than checking all these conditions here.